apps/portals/nf/src/pages/DatasetDetailsPage.tsx

```diff
@@ -40,6 +40,7 @@ export default function DatasetDetailsPage() {
         ContainerProps={{
           maxWidth: 'xl',
         }}
+        resourcePrimaryKey={['id']}
```

Comment:
Identifies which searchParams should be used for the canonical URL. This is necessary because some details pages use multiple keys to show a resource. Without this parameter, a canonical URL will not be added (so this must be manually curated for each DetailsPage)